markdown/reference/sql/CREATE-CONVERSION.html.md.erb

```diff
@@ -0,0 +1,72 @@
+---
+title: CREATE CONVERSION
+---
+
+Defines a new encoding conversion.
+
+## <a id="topic1__section2"></a>Synopsis
+
+``` pre
```

Comment:
Global:  Need to add < and > around variables like \<name\> \<source_encoding\> \<dest_encoding\> \<funcname\>